webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -56,10 +73,57 @@ class SubscriptionsPage extends Component {
     this.props.loadSubscriptions();
   }
 
+  handleDoneTask(taskToPoll) {
```

Comment:
Should this live in SubscriptionPage? This seems like something we would want on other pages. Maybe it can be abstracted out?

Reply:
I had planned to abstract it out once we had another usage.  I try to be careful to have at least two usages before refactoring something out into an abstraction so as to make sure I consider multiple use cases.

> @waldenraines would setting min-height for the modal container help?

Maybe, I can add that and see if it helps.

Reply:
I would rather things be broken up now, as it generally gets harder to do later and developers are less likely to break things into more granular pieces as time goes on, then you wind up with large monolithic pieces. This is just my two cents, I leave it up to your judgment and don't consider this a blocker to the PR. :)

Reply:
Updated.